src/main/java/TestFile.java

```diff
@@ -13,22 +13,22 @@ private static void incorrectAssignmentInIfCondition() {
     }
 
     private static void incorrectComparingToItself() {
-        int x = 3423421;
-        if (x == x) {
+        int x = 3421;
```

Comment:
[Checkstyle] WARNING: 'member def type' have incorrect indentation level 8, expected level should be 4.